### PR TITLE
libical

### PR DESCRIPTION
**Pull request by scopatz**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by scopatz** [comment]
@conda-forge/staged-recipes - this is ready for review!

File: recipes/libical/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - cmake >=3.11.0
+    - perl
+    - pkg-config  # [osx]
+  host:
+    - icu
+    - glib >=2.32
```
Comment (isuruf) [inline recipes/libical/meta.yaml]: Remove the version constraint here so that pinning works.

File: recipes/libical/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    - libxml2 >=2.7.3
```
Comment (isuruf) [inline recipes/libical/meta.yaml]: Same here

File: recipes/libical/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    - gobject-introspection >=0.6.7  # [linux]
+  run:
+    - icu
+    - glib >=2.32
+    - libxml2 >=2.7.3
+    - gobject-introspection >=0.6.7  # [linux]
```
Comment (isuruf) [inline recipes/libical/meta.yaml]: Does this need a pinning to minor version?

File: recipes/libical/meta.yaml (hunk 1 above)
Comment (scopatz) [reply recipes/libical/meta.yaml]: I am not sure I understand.  Can you elaborate?

File: recipes/libical/meta.yaml (hunk 3 above)
Comment (scopatz) [reply recipes/libical/meta.yaml]: This isn't a hard pin.  This is what the docs say the version constraint is for all of these packages

File: recipes/libical/meta.yaml (hunk 1 above)
Comment (scopatz) [reply recipes/libical/meta.yaml]: do you mean the pins coming from run_exports in those packages?

Comment (isuruf) [reply recipes/libical/meta.yaml]: I mean pins coming from `conda-forge-pinning`'s `conda_build_config.yaml`

Comment (isuruf) [reply recipes/libical/meta.yaml]: In `conda_build_config.yaml` if you have,
```yaml
glib:
  - 2.56
```

then, following,
```yaml
host:
  - glib
run:
  - glib
```

will be rendered as
```yaml
host:
  - glib  2.56.2
run:
  - glib >=2.56.2,<2.57.0.a0
```


following,
```yaml
host:
  - glib >=2.32
run:
  - glib >=2.32
```

will be rendered as
```yaml
host:
  - glib  2.58.3
run:
  - glib >=2.58.3,<2.59.0a0
```

File: recipes/libical/meta.yaml (hunk 3 above)
Comment (isuruf) [reply recipes/libical/meta.yaml]: I mean, you should have
```yaml
- {{ pin_compatible("gobject-introspection", max_pin="x.x") }}
```

**Comment by scopatz** [comment]
@conda-forge/staged-recipes - pinging this for review again

**Pull request merged**